Lib/test/test_locale.py

```diff
@@ -555,6 +565,7 @@ def test_getpreferredencoding(self):
             # If encoding non-empty, make sure it is valid
             codecs.lookup(enc)
 
+    @unittest.skip("TODO: RustPython strcoll has not been implemented")
```

Comment:
Does this work?

```suggestion
    # TODO: RUSTPYTHON, strcoll has not been implemented
    @unittest.expectedFailure
```

Reply:
Sorry! I accidentally skip it since I haven't implemented the strcoll but forgot that it will raise error nonetheless :((

Reply:
That particular test is expecting to raise a `TypeError` there. If it doesn't, it fails.